bin/spring

```diff
@@ -1,15 +1,17 @@
 #!/usr/bin/env ruby
 
-# This file loads spring without using Bundler, in order to be fast.
+# This file loads Spring without using Bundler, in order to be fast.
 # It gets overwritten when you run the `spring binstub` command.
 
 unless defined?(Spring)
-  require "rubygems"
-  require "bundler"
+  require 'rubygems'
+  require 'bundler'
```

Comment:
Do you have a linter doing some stuff? Or are you a... 😎 single man, now?

Reply:
This is 100% Spring: I ran `spring binstub rspec` and this is what it generated.